ordersettler/ordersettler.go

```diff
@@ -137,26 +137,11 @@ func (r *OrderSettler) findNewSettlements(ctx context.Context) error {
 			}
 
 			// ensure order exists on source chain
-			exists, err := sourceBridgeClient.OrderExists(ctx, sourceGatewayAddress, fill.OrderID, big.NewInt(int64(height)))
+			exists, amount, err := sourceBridgeClient.OrderExists(ctx, sourceGatewayAddress, fill.OrderID, big.NewInt(int64(height)))
 			if err != nil {
 				return fmt.Errorf("checking if order %s exists on chainID %s: %w", fill.OrderID, sourceChainID, err)
 			}
 			if !exists {
-				// if the order does not exist on the source, set the
```

Comment:
The orders table should be considered logically separate from the order_fills table. There may not be an orders entry for every order_fills entry.